Automattic/jetpack
# VideoPress: Fix icon on privacy column

Fixes #27161

#### Changes proposed in this Pull Request:

* Removes `fill="none"` from local icons
* Fixes privacy check needed to show icon

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Go to the VideoPress dashboard with at least one video
* Change the view to List instead of Grid
* Change the privacy of a video to private
* Check that the icon now is visible

![2022-11-04_17-29-10](https://user-images.githubusercontent.com/8486249/200069391-a432a888-4928-4c33-87a8-8640008b4d27.png)
